include/seqan3/alphabet/nucleotide/dna4.hpp

```diff
@@ -256,6 +329,9 @@ struct dna4
         }()
     };
 
+    //!\brief The complement table.
+    static const std::array<dna4, value_size> complement_table;
```

Comment:
mhmm, we don't have some rules for this in the core guidelines, but I think that is bad style.
I mean you just declare it `static const` here and later one use `constexpr` in the definition, only to put the initialisation out of the class definition, while instead you could directly declare the value `constexpr`.